windows_containers/windows-containers-release-notes-3-x.adoc

```diff
@@ -0,0 +1,103 @@
+
+The release notes for Red Hat OpenShift for Windows Containers tracks the development of the WMCO, which provides all Windows container workload capabilities in {product-title}.
+
+[id="getting-support"]
+== Getting support
+
+You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
+
+[id="wmco-3-0-0"]
+== Release notes for Red Hat Windows Machine Config Operator 3.0.0
+
+This release of the WMCO provides bug fixes and enhancements for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 3.0.0 were released in link:https://access.redhat.com/errata/RHBA-2021:xxxx[RHBA-2021:xxxx].
+
+// Replace above advisory link when known (available from Errata tool once live ID is assigned)
+
+WMCO supports self-managed clusters built using installer-provisioned infrastructure running on the following cloud providers:
+
+// The content below may require updates based on BYOH epic.
+
+* Amazon Web Services (AWS)
+* Microsoft Azure
+* VMware vSphere
+
+The following Windows Server operating systems are supported in this release of the WMCO, depending on which platform your cluster is installed on:
+
+[cols="1,4",options="header"]
+|===
+
+|WMCO platform
+|Windows Server version
+
+|AWS
+|Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019
+
+|Azure
+|Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019
+
+|vSphere
+|Windows Server Semi-Annual Channel (SAC): Windows Server 2004 with link:https://support.microsoft.com/en-us/topic/august-11-2020-kb4566782-os-build-19041-450-ec5cf4b6-86ed-b0d3-539c-5fbe1b17f56a[Microsoft patch KB4566782]
+
+|===
+
+[IMPORTANT]
+====
+Running Windows container workloads is not supported for clusters in a restricted network or disconnected environment.
+====
+
+Version 3.x of the WMCO is only compatible with {product-title} 4.8.
+
+[id="wmco-3-0-0-improvements"]
+=== New features and improvements
+
+This release adds the following new features and improvements.
+
+// Addresses Github #32620 - https://github.com/openshift/openshift-docs/pull/32620
+==== Clarified limits on custom VXLAN port selection
+
+Users may not select a custom VXLAN port when using the latest version of Windows server.
+
+// Addresses WINC-551 - https://github.com/openshift/windows-machine-config-operator/pull/279
+==== Added support for autoscaling Windows nodes
+
+After defining and deploying a ClusterAutoscaler, MachineSet, and a MachineAutoscaler, then
+// not sure what else goes here
```

Comment:
Definitely, but I guess my biggest question is why would a customer want to use this functionality. It can be very high level and just a single sentence. I'm not sure I understand it enough to answer that though.

Reply:
Cluster autoscaling is a provider-agnostic solution to dynamic computing resource requirements. Cluster autoscaling automatically scales up or down the number of machines cluster to meet its current deployment needs, keeping workloads available and pruning unused resources when possible.